kfs-cam/src/main/java/org/kuali/kfs/module/cam/businessobject/AssetBase.java

```diff
@@ -151,7 +152,9 @@
     protected ObjectSubType financialObjectSubType;
     protected AssetAcquisitionType acquisitionType;
     protected ContractsAndGrantsAgency agency;
+    protected List<Note> boNotes = new ArrayList<Note>();

```

Comment:
Checked with Matt. It looks like notes don't get deleted from an Asset, and Assets aren't deleted (only inactivated)